README_SETUP.md

```diff
@@ -37,9 +37,9 @@ choco install <package_name>
 ```
 </p></details>
 
-### Required Software - Java JDK 17
+### Required Software - Java JDK 21
 
-Download and install the latest JDK 17 (17.0.5 or later) release from either of these locations:
+Download and install the latest JDK 21 (21.0.5 or later) release from either of these locations:
 
 * [Adoptium/Temurin](https://github.com/adoptium/temurin17-binaries/releases) - The Adoptium Working Group promotes and supports high-quality runtimes and associated technology for use across the Java ecosystem
```

Comment:
wrong link, it should be 
`https://github.com/adoptium/temurin21-binaries/releases`